source/server/options_impl.cc

```diff
@@ -11,6 +11,10 @@
 #include "spdlog/spdlog.h"
 #include "tclap/CmdLine.h"
 
+#if defined(__APPLE__)
+#define uint64_t unsigned long
```

Comment:
This should be in header `<cstdint>` included above. You might need to do `using std::uint64_t;` however.